AccountService.js

```diff
@@ -40,4 +40,72 @@ export class AccountService {
     });
     return response.data;
   }
+
+  /**
+   * @method accounts
+   * @param {Object} config
+   * @param {string} config.url the base url
+   * @param {string} config.email the user's email
+   * @param {string} config.after an account's DID
+   * @param {number} config.limit how many accounts to return
+   * @return {Array} data
+   * @description calls on the adminr route and returns all
```

Comment:
typo

Reply:
wrt jsdocs, we have been using the hyphenated style as shown here.  @dlongley @davidlehn @gannan08 anyone have opinions about this?

http://usejsdoc.org/tags-param.html

Reply:
I will use the hyphen although the config.after is just there to describe properties of the object being passed in.